drivers/mmcsd/mmcsd_sdio.c

```diff
@@ -1946,6 +1996,23 @@ static ssize_t mmcsd_writemultiple(FAR struct mmcsd_state_s *priv,
         }
     }
 
+#ifdef CONFIG_MMCSD_MMCSUPPORT
```

Comment:
Remove the ifdef else here too.

Reply:
yes, we don't need both compiler time and run time check, let's remove CONFIG_MMCSD_MMCSUPPORT and always use runtime check.

Reply:
ditto